templates/location.html

```diff
@@ -46,20 +41,20 @@ <h1>Location - {{ locdet["name"] }}</h1>
 {% set rack_num = "{:02d}".format(rack_id| int)|string %}
 
 {% if rack_id != none %}
-	<h2>Current Rack: {{ rack_num }}</h2>
+	<h2>Current Rack: {{loc_id}}-{{ rack_num }}</h2>
 {% endif %}
 
-<table class="table table-bordered border-dark" id="cellTable">
+<table class="table table-bordered border-dark table-fixed" id="cellTable">
 	<thead>
 		<tr class="table-primary">
 			<th scope="col">#</th>
 			{% for row in range(columns) %}
                 {% if rack_id == none %}
-                    <th scope="col" >Rack {{ loop.index }}</th>
+                    <th scope="col">Rack {{ "{:02d}".format(loop.index) }}</th>
 				{% elif rack_id | int % 2 == 1 %}
-					<th scope="col" >Column {{ loop.index }}</th>
+					<th scope="col" style="text-align: center;">Cell {{ "{:02d}".format(loop.index) }}</th>
 				{% else %}
-					<th scope="col" >Column {{ loop.revindex }}</th>
+					<th scope="col" style="text-align: center;">Cell {{ "{:02d}".format(loop.index) }} </th>
```

Comment:
Check bootstrap documentation for _text-centre_ class or something similar, theres a class for centering text. Also, it's bad practice to write CSS directly into the template.